File: Java/CreditCard.java
import java.sql.Date;

class CreditCard extends Payment{
    Integer cardNumber;
    Integer cvv;
    Date expirationDate;
    Integer ammount;

    public CreditCard(Integer id, Integer cardNumber, Integer cvv, Date expirationDate, Integer ammount){
        super(id);
        this.cardNumber = cardNumber;
        this.cvv = cvv;
        this.expirationDate = expirationDate;
        this.ammount = ammount;
        
    }
}

File: JS/CreditCard.js
class CreditCard extends Payment {
    constructor(id, cardNumber, cvv, expirationDate, ammount){
    super(id)
    this.paypalAccount = paypalAccount;
    this.cardNumber = cardNumber;
    this.cvv = cvv;
    this.ammount = ammount;
    }
}